package.json

```diff
@@ -59,32 +61,37 @@
     "solc0_8": "npm:solc@^0.8.6",
     "sqlite3": "^5.0.2",
     "twig": "^1.15.4",
+    "typescript": "^4.5.5",
     "uuid": "^8.3.2",
     "web3": "^1.5.2",
     "web3-provider-engine": "16.0.3",
     "web3-providers-http": "1.6.0",
     "websocket": "^1.0.34"
   },
   "devDependencies": {
-    "@babel/eslint-parser": "^7.17.0",
+    "@typescript-eslint/eslint-plugin": "^5.12.0",
+    "@typescript-eslint/parser": "^5.12.0",
     "eslint": "^8.9.0",
     "eslint-plugin-filenames": "^1.3.2",
     "eslint-plugin-prefer-arrow": "^1.2.3",
     "jest": "^27.0.6",
     "nodemon": "^2.0.12",
     "sequelize-cli": "^6.2.0",
-    "solhint": "^3.3.7"
+    "solhint": "^3.3.7",
+    "ts-node": "^10.5.0"
   },
   "scripts": {
     "test": "jest",
     "test:db:docker": "printf '\\e[2J\\e[3J\\e[H' && npm run migrate && NODE_ENV=development npm run test",
     "test:restart": "docker-compose -f docker-compose.test.yaml restart tests && docker logs -f pointnetwork_tests",
-    "start": "./point",
-    "start:docker": "npm run compile && npm run migrate && npm start",
-    "start:docker:e2e": "npm run migrate && MODE=e2e npm start",
+    "start": "node dist/index.js",
+    "start:docker": "npm run compile && npm run migrate && npm run build && npm start",
+    "start:docker:dev": "npm run compile && npm run migrate && npm run start:dev",
+    "start:docker:e2e": "npm run migrate && npm run build && MODE=e2e npm start",
     "start:dev": "nodemon",
-    "migrate": "./point migrate",
-    "compile": "./point compile",
+    "build": "tsc && cp -R src/truffle dist/truffle && cp -R src/internal dist/internal && cp -R src/client/proxy/certs dist/client/proxy/certs && cp src/client/proxy/ca.srl dist/client/proxy/ca.srl",
```

Comment:
We won't need to copy these dirs to `dist` if we don't store them in src as suggested in the above comment. Also certs should probably be placed to resources I think.